src/styles/reset.scss

```diff
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
```

Comment:
You can add rules for font to the html tag selector. Also reset margin for body down, where you reset styles for p tag using comma.